meterfs/test_meterfs_openclose.c

```diff
@@ -0,0 +1,96 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Meterfs opening and closing tests group
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Tomasz Korniluk
+ *
+ *
+ * %LICENSE%
+ */
+
+#include "common.h"
+
+static struct {
+	int fds[255];
+	char buff[21];
+}common;
+
+TEST_GROUP(meterfs_openclose);
+
+
+TEST_SETUP(meterfs_openclose)
+{
+	memset(common.buff, 0, sizeof(common.buff));
+}
+
+
+TEST_TEAR_DOWN(meterfs_openclose)
+{
+	TEST_ASSERT_EQUAL(0, file_eraseAll());
+}
+
+
+/* test case of opening and closing non existing files */
+TEST(meterfs_openclose, no_files)
+{
+	int i;
+
+	for (i = 0; i < 255; ++i) {
```

Comment:
Maybe use `sizeof(common.fds) / sizeof(common.fds[0])` instead of magic number? Applies to all `for` loops

Reply:
Done.